### PR TITLE
[4.4] Make UsersLatestHelper backward compatible

### PR DESCRIPTION
**Pull request by joomdonation**
Pull Request for Issue # .

### Summary of Changes
This is a follow up for PR https://github.com/joomla/joomla-cms/pull/40092 to make it backward compatible. Can be merged by code review @laoneo . See https://github.com/joomla/joomla-cms/pull/40092#issuecomment-1473878737 for information

### Testing Instructions
Code review

### Actual result BEFORE applying this Pull Request
Works, but not backward compatible

### Expected result AFTER applying this Pull Request
Works.

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by laoneo** [comment]
Thanks, can you add the deprecated version like we do in https://github.com/joomla/joomla-cms/pull/38657. And the target version should be 6.0.

**Comment by joomdonation** [comment]
> Thanks, can you add the deprecated version like we do in #38657. And the target version should be 6.0.

Done.

**Comment by sandewt** [comment]
Thks

File: modules/mod_users_latest/src/Helper/UsersLatestHelper.php [hunk 1]
```diff
@@ -74,4 +75,24 @@ public function getUsers(Registry $params, SiteApplication $app): array
...
+     *             Use the none static method getLatestUsers
+     *             Example: Factory::getApplication()->bootModule('mod_users_latest', 'site')
+     *                          ->getHelper('UsersLatestHelper')
+     *                          ->getLatestUsers($params, Factory::getApplication())
+     */
+    public static function getUsers($params)
+    {
+        return (new self())->getLatestUsers($params, Factory::getApplication());
+    }
 }
```
Comment (sandewt) [inline modules/mod_users_latest/src/Helper/UsersLatestHelper.php:98]: Some `modules` have already been modified in the past. I assume that the depreated text of the static method also needs to be adjusted.

To know: `@deprecated 5.0 Use the none static function getArticles` to `@deprecated __DEPLOY_VERSION__ will be removed in 6.0` and so on.

It concerns the following helper files:
`ArticlesLatestHelper.php`, see link: https://github.com/joomla/joomla-cms/blob/ef06fa21435a4937b7df19d084f35affa517e677/modules/mod_articles_latest/src/Helper/ArticlesLatestHelper.php#L159
`ArticlesNewsHelper.php`
`ArticlesPopularHelper.php`

If yes, then a new PR is needed.

Comment (joomdonation) [reply modules/mod_users_latest/src/Helper/UsersLatestHelper.php:98]: @sandewt  Yes. Please make PR to change deprecated note for these files.

Comment (sandewt) [reply modules/mod_users_latest/src/Helper/UsersLatestHelper.php:98]: See pr #40156

**Pull request merged**